themes/default/content/blog/deploy-ai-ml-vercel-app/index.md

```diff
@@ -0,0 +1,201 @@
+---
+date: 2023-09-11
+title: "Deploying Your AI/ML Chatbot Frontend To Vercel Using Pulumi"
+allow_long_title: true
+meta_desc: "Learn how to effortlessly deploy your AI/ML chatbot frontend to Vercel using Pulumi and Python to take your AI DevOps skills to the next level"
+meta_image: meta.png
+authors:
+    - adora-nwodo
+tags:
+    - ai
+    - ml
+    - iac
+    - vercel
+    - vuejs
+    - chatbot
+
+---
+
+The process of taking an idea and turning it into reality has been nothing short of extraordinary since we started innovating with Artificial Intelligence. With this technology, machines learn about and communicate with people, while also helping us in ways we never could have imagined only a few years ago. If you've been following along, you might recall our [previous article](https://www.pulumi.com/blog/mlops-the-ai-challenge-is-cloud-not-code/) where we used Python and Pulumi to a chatbot API (named katwalk) to the cloud.
+
+But our AI adventure hasn't ended there; it's only just begun.
+
+Now, imagine this: A Vue.js chatbot frontend, seamlessly integrating human interaction and the AI chatbot API. This blog post is part of the series on AI Infrastructure as Python. We'd delve into the creation of a captivating Vue.js chatbot frontend, and with Pulumi, we'll deploy it to Vercel.
+
+Before we get into the details of using Pulumi to deploy a Vue.js frontend app to Vercel, let's start with some quick terminology/context:
+
+- Katwalk Frontend is a demo Vue.js application that integrates with the [Katwalk Server](https://github.com/pulumiverse/katwalk) to produce a function web interface for you to communicate with your chatbot. 
+- [Vercel](https://vercel.com/) is a modern web development cloud platform and hosting provider. It specialises in making web application deployment simple and efficient.
+
+## Integrate Your Chatbot Service With a Frontend And Deploy It!
```

Comment:
Ok, I think we are missing a critical piece which is why do we care to use Pulumi to do this? What is the problem right now? 

I think we should lay out how building an end to end AIML app is actually hard because there are so many pieces of infrastructure. We want to coordinate the programmatic creation of all the infrastructure from the frontend to backend. So having scripts or manually doing it is bad, so having Pulumi do it all for you is better for developer productivity, better for scalability, etc, etc